spec/rspec/mocks/partial_double_spec.rb

```diff
@@ -377,6 +377,17 @@ def defined_private_method
         expect(object.send(:defined_private_method)).to eq("works")
       end
 
+      it 'can be temporarily supressed' do
+        without_verifying_partial_doubles do
+          expect(object).to receive(:fictitious_method) { 'works' }
+        end
+        expect(object.fictitious_method).to eq 'works'
+
+        expect {
+          expect(object).to receive(:another_fictitious_method) { 'works' }
+        }.to raise_error RSpec::Mocks::MockExpectationError
+      end
```

Comment:
As I mentioned above, I think there's a bug in your implementation that makes `without_verifying_partial_doubles` apply to normal doubles in addition to partial doubles.  Regardless, it would be nice to have some specs document how normal doubles work in a `without_verifying_partial_doubles` (e.g. they should work exactly the same).  Maybe I'm wrong and a spec will show that I'm wrong, or maybe I'm right and a spec will confirm that.